pdf-service/package.json

```diff
@@ -64,7 +64,8 @@
     "eslint": "^3.1.1",
     "mustache": "^3.0.1",
     "nodemon": "^1.19.4",
-    "pdfmake": "^0.1.62"
+    "pdfmake": "^0.1.62",
+    "node-fetch": "^2.6.1"
```

Comment:
Don't add a new dependency, remove this and use `axios` instead

Reply:
removed unnecessary dependency and use axios for https request call